app/src/main/java/com/example/kuba10/mypokemonplaces/ListFragmentAdapters/AdapterForTouchAndFirebase.java

```diff
@@ -1,186 +0,0 @@
-package com.example.kuba10.mypokemonplaces.ListFragmentAdapters;
-
-import android.view.MotionEvent;
-import android.view.View;
-import android.widget.ImageView;
-
-import com.example.kuba10.mypokemonplaces.Constants;
-import com.example.kuba10.mypokemonplaces.ListFragment.FirebaseListFragment;
-import com.example.kuba10.mypokemonplaces.Model.PokePlace;
-import com.example.kuba10.mypokemonplaces.Model.PokemonGo;
-import com.example.kuba10.mypokemonplaces.PlaceDetailsFragment.PlaceDetailsFragment;
-import com.example.kuba10.mypokemonplaces.R;
-import com.firebase.ui.database.FirebaseRecyclerAdapter;
-import com.google.firebase.database.ChildEventListener;
-import com.google.firebase.database.DataSnapshot;
-import com.google.firebase.database.DatabaseError;
-import com.google.firebase.database.DatabaseReference;
-import com.google.firebase.database.Query;
-import com.squareup.picasso.Picasso;
-
-import java.util.ArrayList;
-import java.util.Collections;
-
-public class AdapterForTouchAndFirebase extends FirebaseRecyclerAdapter<PokePlace, CardViewHolder> implements ItemTouchHelperAdapter {
-
-    private final Query ref;
-    private ChildEventListener mChildEventListener;
-    private ArrayList<PokePlace> userPlacesDataList;
-    private ArrayList<PokemonGo> pokemonGo_data_list;
-    private FirebaseListFragment parentFragment;
-    private OnStartDragListener mOnStartDragListener;
```

Comment:
To pole może być `final` + unikaj notacji węgierskiej (bez tego m na początku). Bóg (:smile:) mówi czemu: http://jakewharton.com/just-say-no-to-hungarian-notation/

Reply:
ciekawe, masa źródeł używa tej formy - ale dobrze wiedzieć